src/components/link-preview/link-preview.pw.tsx

```diff
@@ -0,0 +1,258 @@
+  test("should verify hover color of Link Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest />);
+
+    const previewElement = linkPreview(page);
+    await previewElement.hover();
+
+    await expect(previewElement).toHaveCSS(
+      "background-color",
+      "rgb(204, 214, 219)"
+    );
+  });
+
+  test("should verify border outline color and width of Link Preview on focus", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(<LinkPreviewComponentTest />, {
+      hooksConfig: { focusRedesignOptOut: true },
+    });
+
+    await linkPreview(page).focus();
+
+    await checkGoldenOutline(linkPreview(page), "2px");
+  });
+
+  test("should verify border outline color of close icon on focus", async ({
+    mount,
+    page,
+  }) => {
+    await mount<HooksConfig>(
+      <LinkPreviewComponentTest as="div" onClose={() => {}} />,
+      {
+        hooksConfig: { focusRedesignOptOut: true },
+      }
+    );
+
+    const closeButton = page.getByLabel("link preview close button");
+
+    await closeButton.focus();
+    await checkGoldenOutline(closeButton);
+  });
+});
+
+test.describe("check events for Link Preview component", () => {
+  test("should call onClose callback when a click event is triggered", async ({
+    mount,
+    page,
+  }) => {
+    let callbackCount = 0;
+    await mount(
+      <LinkPreviewComponentTest
+        as="div"
+        onClose={() => {
+          callbackCount += 1;
+        }}
+      />
+    );
+
+    await linkPreviewCloseIcon(page).click();
+
+    expect(callbackCount).toEqual(1);
+  });
+
+  keysToTrigger.forEach((key) => {
+    test(`should call onClose callback when a keyboard key ${key} event is triggered`, async ({
+      mount,
+      page,
+    }) => {
+      let callbackCount = 0;
+
+      await mount(
+        <LinkPreviewComponentTest
+          as="div"
+          onClose={() => {
+            callbackCount += 1;
+          }}
+        />
+      );
+
+      await page.keyboard.press("Tab");
+      await page.keyboard.down(key);
+
+      await linkPreviewCloseIcon(page).waitFor();
+      expect(callbackCount).toEqual(1);
+    });
+  });
+});
+
+test.describe("Accessibility tests for LinkPreview component", () => {
+  test("should pass accessibilty tests for default story", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest />);
+
+    await checkAccessibility(page);
+  });
+
+  test("should pass accessibility tests for loading state", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest isLoading />);
+
+    await checkAccessibility(page);
+  });
+
+  test("should pass accessibility tests for close icon", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkPreviewComponentTest onClose={() => {}} />);
+
+    await checkAccessibility(page);
+  });
+});
+
+test("should render with the expected border radius styling", async ({
```

Comment:
**suggestion: (non blocking)** Would you be able to move this in to the main block with the rest of the tests? Seems a bit odd to have it after the accessibility tests. I know you've copied this from the Cypress tests as that's what we've done there but it might be nice to tidy them up in to the main block.

Reply:
@DipperTheDan I don't think that test case should be in with any of the other describes, unless those others relating to things like border outline colours etc are put in their own `describe` for, say, styling. This is liked with our discussion in the channel about test cases being out on their own or in a `describe` where we decided if they're not related to the other cases in the existing `describe` blocks then they can remain out on their own.

Reply:
@DipperTheDan I've since decided that we should wrap all of our tests in a `describe` regardless of number to keep the format and standard uniform throughout :)